manual/tracy.tex

```diff
@@ -1519,6 +1519,8 @@ \subsubsection{Vulkan}
 
 You also need to periodically collect the GPU events using the \texttt{TracyVkCollect(ctx, cmdbuf)} macro\footnote{It is considerably faster than the OpenGL's \texttt{TracyGpuCollect}.}. The provided command buffer must be in the recording state and outside a render pass instance.
 
+For cases where you are not static linking the Vulkan loader, you can use TRACY_VK_USE_SYMBOL_TABLE to have the underlying implementation dynamically load the Vulkan symbols needed for sampling.
```

Comment:
You are modifying existing interfaces. Describe how the new approach changes what you need to pass to macros, preferably with an example that shows how things look in both approaches.

Use proper formatting for fixed-width text. The macro name must be properly escaped to work as intended.